README-fi.md

```diff
@@ -0,0 +1,24 @@
+# 6Dash
+API ja optimointi modi [3Dash](https://delugedrop.itch.io/3dash):ille
```

Comment:
is `:ille` supposed to be here?

Reply:
Yes, you can delete the : but it makes it more clear